### PR TITLE
Prepare for CI

### PR DESCRIPTION
**Pull request by oribon**
Added required steps to enable ci-operator:
* Added simple validation suite from cnf-features-deploy
* Created bundleci.Dockerfile which will be consumed by ci-operator

### CODE REVIEW

File: test/utils/client/clients.go [hunk 1]
```diff
@@ -5,14 +5,17 @@ import (
 
 	"github.com/golang/glog"
 
+	ptpv1 "github.com/openshift/ptp-operator/pkg/client/clientset/versioned/typed/ptp/v1"
```
Comment (SchSeba) [inline test/utils/client/clients.go:8]: nit: move this one back to the last part please :)

File: Makefile [hunk 2]
```diff
@@ -217,7 +217,10 @@ clean:
 	rm -rf build/_output/bin/ptp-operator
 
 functests:
-	hack/run-functests.sh
+	SUITE=./test hack/run-functests.sh
```
Comment (SchSeba) [inline Makefile:220]: please move the older tests to conformance

like

test/
-> conformance
--> ptp

File: test/utils/client/clients.go (hunk 1 above)
Comment (oribon) [reply test/utils/client/clients.go]: done

File: Makefile (hunk 2 above)
Comment (oribon) [reply Makefile]: done

**Review by SchSeba — APPROVED** [review]
/lgtm
/approve

**Pull request merged**